RELEASE.md

```diff
@@ -0,0 +1,15 @@
+- Check out the repository and pull the latest updates on MASTER (origin or your fork should work)
+- Update the package version in `package.json`.
+- Update `CHANGELOG.md` - remember to update the links at the bottom of the document.
+- Run `npm install`
```

Comment:
did you mean `npm version`?

Reply:
> One time I did `npm version` and then later `gh-release` failed

now that I think about it, that actually makes sense. `gh-release` helps with creating a tag (and release) on GitHub using the information in the CHANGELOG. it expects that you will have bumped the version number in your package.json file manually, but _not_ already tagged a version.

`npm version` bumps your package.json and creates a (local) tag.